core/src/main/java/bio/terra/pearl/core/model/audit/ResponsibleEntity.java

```diff
@@ -33,4 +34,10 @@ public ResponsibleEntity(String systemProcess) {
         }
         this.systemProcess = systemProcess;
     }
+    public ResponsibleEntity(Boolean anonymousUser) {
```

Comment:
nit: should this just be `ResponsibleEntity()` to lean into the "anonymous"-ness? I would find `new ResponsibleEntity(true)` a bit arcane...